### ISSUE TITLE
2.3.0: Browser integration settings missing

### ISSUE THREAD

**Issue by rask** [keepassxreboot/keepassxc#1642]
I'm running KeepassXC 2.3.0 on an Ubuntu 14.04 machine. KeepassXC was installed via the official PPA.

## Expected Behavior
I open KeepassXC settings and I should see a "Browser integration" option in the settings sidebar.

Both Chromium and Firefox extensions should show `2.3.0` when querying the local KeepassXC installation version. Additionally, they should show that `2.3.0` is the latest available version.

## Current Behavior
I open KeepassXC settings and I do not see a "Browser integration" option in the settings sidebar. Only "Legacy browser integration" is visible.

Both Chromium and Firefox extensions show `unknown` when querying the local KeepassXC installation version. They do show `2.3.0` as the available latest downloadable version.

## Steps to Reproduce (for bugs)
1. Install KeepassXC from PPA on Ubuntu 14.04
2. Open settings and check if "Browser integration" is available
3. Check locally installed version on the Chromium and Firefox extensions settings pages.

## Context
Would like to manage and use secrets when using a web browser. Currently impossible.

## Debug Info
KeePassXC - 2.3.0

Libraries: not sure

Operating system: Ubuntu 14.04
CPU architecture: 64bit
Kernel: 4.4.0

Enabled extensions:
- Chromium and Firefox extensions

**Comment by phoerious** [comment]
Browser needs Qt 5.3, so it's not supported on Trusty. Use the AppImage instead.
We should add that to the FAQ.

**Comment by rask** [comment]
You mentioned this on my other issue yes. I need to see how AppImages work. Thanks.

**Comment by phoerious** [comment]
I added an FAQ entry about KeePassXC-Browser on Trusty: https://keepassxc.org/docs/#faq-browser-trusty

**Comment by KitchM** [comment]
Okay, I found the appimage version and downloaded it. I made it executable. I changed the shortcut to use it. I opened it and it worked fine.

Next I opened Firefox Quantum Version 59.0 and got the same message. I removed the add-on. Then I searched for the current add-on and installed it. Then I tried again and got the same message: "KeePassXC-Browser has encountered an error: Timeout or not connected to KeePassXC"

I then opened Settings, and selected the Connected Databases tab. There were none found, so I selected to Connect, hoping that it would allow me to search for the correct one, but that doesn't work either.

One would assume that the database that is recognized by the main program (and is open) would be seen by the browser add-on.
